pkg/interactor/save_data_test.go

```diff
@@ -0,0 +1,95 @@
+package interactor
+
+import (
+	"errors"
+	"testing"
+	"time"
+
+	"github.com/CESARBR/knot-cloud-storage/pkg/entities"
+	"github.com/CESARBR/knot-cloud-storage/pkg/mocks"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	// ErrSaveDataDB is returned when the database fails to save data
+	ErrSaveDataDB = errors.New("error when trying to save data")
+
+	// ErrTokenEmpty is returned for empty access tokens.
+	ErrTokenEmpty = errors.New("no access token provided")
```

Comment:
Yep. They aren't errors that happened in the interaction, they are errors that happened in the storage. For this reason, it would not be good to change the wrapping of errors for static error messages.

No problem @lcbm, I'll change them to lower-case. But anyway, thank you for the review 😆